assets/js/modules/adsense/util/status.js

```diff
@@ -81,7 +83,7 @@ export const determineAccountStatus = ( {
 	}
 
 	if ( undefined === alerts ) {
-		return alertsErrorToStatus( error );
+		return alertsErrorToStatus( alertsError );
```

Comment:
@felixarntz same with this conditional – should it check `alerts` or `alertsError` would be better?

Reply:
Same as above, let's keep it as is.